src/py/_pyodide/_core_docs.py

```diff
@@ -154,7 +154,9 @@ def default_converter(value, convert, cache):
         """
         pass
 
-    def then(self, onfulfilled: Callable, onrejected: Callable) -> "Promise":
+    def then(
+        self, onfulfilled: Callable[..., Any], onrejected: Callable[..., Any]
```

Comment:
I think all of these should be `Callable[Any, Any]` except `create_once_callable` which can be anything.

Eventually it would be nice to offer improved typing support for downstream. Most `JsProxy` objects have different subsets of the methods in this file, along with other methods from JavaScript. Of course the point of this file is primarily to generate docstrings and API docs, the use with mypy is very limited.